app/src/main/java/com/hadiyarajesh/notex/reminder/notification/NotificationBroadCastReceiver.kt

```diff
@@ -26,7 +27,7 @@ class NotificationBroadCastReceiver : HiltBroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
         super.onReceive(context, intent)
         when (intent.action) {
-            context.resources.getString(R.string.done_action) ->
+            context.resources.getString(R.string.complete_action) ->
```

Comment:
better we can use `ContextCompat` for resources. Here we are also having the context for reference.